source/clear-linux/tutorials/multi-boot/multi-boot.rst

```diff
@@ -228,40 +230,34 @@ Install the Clear Linux OS
 
    .. figure:: figures/multi-boot-10.png
 
-      Figure 10: |CL|: Set mount points
+      Figure 10: |CL|: Set mount points.
 
 #. On the :guilabel:`User configuration` screen, select
-   :guilabel:`Create an administrative user`. See Figure 11.
+   :guilabel:`Create an administrative user`, as shown in Figure 11.
 
    .. figure:: figures/multi-boot-11.png
 
-      Figure 11: |CL|: User configuration
+      Figure 11: |CL|: User configuration.
 
-#. Select :guilabel:`Add user to sudoers?`. See Figure 12.
+#. Select :guilabel:`Add user to sudoers?`, as shown in Figure 12.
 
    .. figure:: figures/multi-boot-12.png
 
-      Figure 12: |CL|: Add user as sudoer
+      Figure 12: |CL|: Add user as sudoer.
 
-#. Follow the remaining prompts to complete the installation and go through
-   the out-of-box-experience of |CL|.
+#. Follow the remaining prompts to complete the installation and finish
+   the out-of-box-experience for |CL|.
 
 #. Log in.
 
-#. Get root privileges.
-
-   .. code-block:: console
-
-      $ sudo -s
-
-#. Add a timeout period for Systemd-Boot to wait, otherwise it will not
-   present the boot menu and will always boot |CL|.
+#. Add a Systemd-Boot timeout period or Systemd-Boot will not
+   present the boot menu and will boot |CL|.
```

Comment:
Change "present the boot menu and will boot |CL|" to "present the boot menu of available OSes to choose from and will always boot |CL|."